ui/raidboss/data/05-shb/eureka/delubrum_reginae_savage.txt

```diff
@@ -0,0 +1,159 @@
+# TODO
+2000.0 "--sync--"
+
+### Trinity Seeker
+# -p 5AD3:4011.4 5A98:4079.5
+# -ii 5B33 5B94 5AB8 5AB9 5ABA 5ABB 5ABC 5AD1 5B2A 5B2B
+4000.0 "--sync--" sync / 00:0839:The Theater of One will be sealed off/ window 10000,0
+4000.0 "--Reset--" sync / 00:0839:The Theater of One is no longer sealed/ window 0,2000 jump 0
+
+# Initial Merciful Air
+4011.4 "Verdant Tempest" sync /:Trinity Seeker:5AD3:/ window 20,20
+4015.6 "--sync--" sync /:Trinity Seeker:5A9A:/
+4020.0 "Manifest Avatar" sync /:Trinity Seeker:5ADA:/
+4028.2 "First Mercy" sync /:Trinity Seeker:5B61:/
+4031.4 "Second Mercy" sync /:Trinity Seeker:5B62:/
+4034.6 "Third Mercy" sync /:Trinity Seeker:5B63:/
+4037.8 "Fourth Mercy" sync /:Trinity Seeker:5B64:/
+4040.2 "Mercy Fourfold" #sync /:Trinity Seeker:5B94:/
+4042.1 "Mercy Fourfold" #sync /:Trinity Seeker:5B94:/
+4042.8 "Seasons Of Mercy" sync /:Seeker Avatar:5AC7:/
+4044.0 "Mercy Fourfold" #sync /:Trinity Seeker:5B94:/
+4045.9 "Mercy Fourfold" #sync /:Trinity Seeker:5B94:/
+4047.3 "Merciful Breeze" sync /:Trinity Seeker:5AC8:/
+4048.8 "Merciful Moon" sync /:Aetherial Orb:5AC9:/
+4051.3 "Merciful Breeze" sync /:Trinity Seeker:5AC8:/
+4054.7 "Merciful Blooms" sync /:Trinity Seeker:5ACA:/
+4062.0 "Verdant Tempest" sync /:Trinity Seeker:5AD3:/
+4074.2 "Merciful Arc" sync /:Trinity Seeker:5AD4:/
+
+# Initial Baleful Air (80% or time push)
+4076.5 "--sync--" sync /:5A98:Trinity Seeker/ window 80,20
+4079.5 "Verdant Path" sync /:Trinity Seeker:5A98:/
+4084.1 "Baleful Swathe" sync /:Trinity Seeker:5AD0:/
+4092.2 "Baleful Onslaught" sync /:Trinity Seeker:5AD5:/
+4099.3 "Phantom Edge?" sync /:Trinity Seeker:5ABD:/
+4105.5 "Baleful Onslaught" sync /:Trinity Seeker:5AD6:/
+4117.8 "--sync--" sync /:Trinity Seeker:5A9A:/
+4121.7 "Scorching Shackle" sync /:Trinity Seeker:5ACB:/
+4126.9 "Baleful Blade" sync /:Trinity Seeker:5ABE:/
+4132.1 "Manifest Avatar" sync /:Trinity Seeker:5ADA:/
+4138.8 "Baleful Comet" sync /:Seeker Avatar:5AD7:/
+4139.8 "Baleful Comet" sync /:Seeker Avatar:5AD7:/
+4140.8 "Baleful Comet" sync /:Seeker Avatar:5AD7:/
+4141.8 "Baleful Comet" sync /:Seeker Avatar:5AD7:/
+4142.2 "Phantom Edge" sync /:Trinity Seeker:5ABD:/
+4144.4 "--sync--" sync /:Trinity Seeker:5A9A:/
+4146.4 "Baleful Firestorm" sync /:Seeker Avatar:5AD8:/
+4148.4 "Baleful Firestorm" sync /:Seeker Avatar:5AD8:/
+4150.4 "Baleful Firestorm" sync /:Seeker Avatar:5AD8:/
+4152.4 "Baleful Firestorm" sync /:Seeker Avatar:5AD8:/
+4153.5 "Baleful Blade" sync /:Trinity Seeker:5ABF:/
+4162.6 "Verdant Tempest" sync /:Trinity Seeker:5AD3:/
+4171.7 "Verdant Path" sync /:Trinity Seeker:5A99:/
+
+# TODO: in progress
+
+### Stygimoloch Warrior
+# TODO: this and Dahu can be seen from each other.
+# Jump here via injected "1961" auto? or some other ability similar to castrum timeline.
+#6000.0 "--sync--" sync / 00:0839:Pride of the Lioness will be sealed off/ window 10000,0
+#6000.0 "--Reset--" sync / 00:0839:Pride of the Lioness is no longer sealed/ window 0,2000 jump 0
+
+### Dahu
```

Comment:
Here is a 6:21 log in case you've missed it: https://www.fflogs.com/reports/tFXmDTp4dvcZMK9G#fight=8